app/controllers/identity.js

```diff
@@ -0,0 +1,94 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import ENV from 'website-my/config/environment';
+
+const BASE_URL = ENV.BASE_API_URL;
+
+export default class IdentityController extends Controller {
+  @tracked isEditClicked = false;
+  @tracked isGenerateChaincodeClicked = false;
+  @tracked identityURL = ' ';
+  @tracked isChecked = false;
+  @tracked isChaincodeClicked = false;
+  @tracked Chaincode = 'Generate Chaincode';
+  @tracked Code = 'Copy Code';
+
+  @action handleCopy() {
+    navigator.clipboard.writeText(this.Chaincode).then(
+      function () {
+        alert('Copied!');
+      },
+      function (err) {
+        alert('Could not copy code: ', err);
+      }
+    );
+  }
+
+  @action handleVerify() {
+    if (this.isChecked === false) {
+      alert('Please verify!');
+    } else {
+      alert('Approved!');
+    }
+  }
+
+  @action handleChaincode() {
```

Comment:
Tomorrow the APIs hopefully get merged so we can merge this function and below generate chain code function

Reply:
Yeah sure.